hi-ml-azure/.vscode/launch.json

```diff
@@ -0,0 +1,63 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run example script in AzureML",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/src/health_azure/examples/elevate_this.py",
+            "cwd": "${workspaceFolder}/src/health_azure/examples/",
+            "args": [
+                "--message='Hello World :-)'",
+                "--azureml"
+            ],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Setup hi-ml-azure package",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/setup.py",
+            "cwd": "${workspaceFolder}/",
+            "args": [
+                "sdist",
+                "bdist_wheel"
+            ],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Setup hi-ml package",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/hi-ml/setup.py",
+            "cwd": "${workspaceFolder}/hi-ml/",
+            "args": [
+                "sdist",
+                "bdist_wheel"
+            ],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "HI-ML-AZURE Make Tests",
+            "type": "python",
+            "request": "launch",
+            "module": "testazure.test_data.make_tests",
+            "cwd": "${workspaceFolder}/testazure/",
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Example Hyperdrive run in AzureML",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/docs/source/examples/8/sample.py",
+            "cwd": "${workspaceFolder}/docs/source/examples/8/",
```

Comment:
This suggests there is a folder `hi-ml/hi-ml-azure/docs`, but as far as I can see the docs still live at `hi-ml/docs`?

Reply:
correct, thanks